pkgs/top-level/all-packages.nix

```diff
@@ -24082,7 +24082,14 @@ with pkgs;
 
   mueval = callPackage ../development/tools/haskell/mueval { };
 
-  mujoco = callPackage ../applications/science/robotics/mujoco { };
+  mujoco =
+    (import ../applications/science/robotics/mujoco {
+    inherit cmake fetchFromGitHub fetchFromGitLab git libGL stdenv xorg fetchpatch
+      SDL2 glm spdlog cereal_1_3_2 eigen lib fetchPypi abseil-cpp python3;
+    inherit (python3Packages) buildPythonPackage setuptools setuptools-scm wheel pybind11;
+    glfw-lib = glfw;
+    glfw = python3Packages.glfw;
+    }).mujoco;
```

Comment:
How can I simplify this? An argument-less `callPackage` did not work as expected.